bin/version-check.js

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env node
+
+/**
+ * Checks the user's node and npm versions to make sure the major
+ * versions match. This script will exit with a non-zero status
+ * (causing the build to fail) if the major version of `node` and
+ * `npm` don't match.
+ * See: https://github.com/Automattic/wp-calypso/pull/22237
```

Comment:
This is what `git blame` is for IMHO

Reply:
Fair enough; this is straightforward anyway and doesn't need the link. I'll remove it 😄